### PR TITLE
Multi db deploy

### PR DESCRIPTION
**Pull request by snopoke**
for deploying with a sharded DB
@czue

### CODE REVIEW

File: corehq/sql_db/management/commands/migrate_multi.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    args = ''
+    help = "Call 'migrate' for each configured database"
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--noinput',
+            action='store_true',
+            dest='noinput',
+            default=False),
+        )
```
Comment (dimagimon) [inline corehq/sql_db/management/commands/migrate_multi.py]: E123 closing bracket does not match indentation of opening bracket's line

File: corehq/preindex/models.py [hunk 2]
```diff
@@ -1,19 +1 @@
...
-        return
-
-    app_name = sender.label.rsplit('.', 1)[0]
-    app_label = app_name.split('.')[-1]
-    plugin = get_preindex_plugin(app_label)
-    if plugin:
-        plugin.sync_design_docs()
-
-signals.pre_migrate.connect(catch_signal)
-signals.post_syncdb.disconnect(syncdb)
```
Comment (snopoke) [inline corehq/preindex/models.py]: This seems to breaks tests

**Comment by czue** [comment]
looks good but not mergeable

**Comment by snopoke** [comment]
:white_check_mark:

**Pull request merged**